### ISSUE TITLE
Incorrect tracker checking

### ISSUE THREAD

**Issue by synctext** [Tribler/tribler#1565]
just move to the next one if it fails?
```
ERROR   1437074811.02         session:259   Tracker[HTTP, http://www.smartorrent.com:2710/ma7697h0anvl19yzswrrdadj57iyt35w/announce] Failed to send HTTP SCRAPE message: [Errno 111] Connection refused
ERROR   1437074881.10         session:259   Tracker[HTTP, http://www.smartorrent.com:2710/ma7697h0anvl19yzswrrdadj57iyt35w/announce] Failed to send HTTP SCRAPE message: [Errno 111] Connection refused
ERROR   1437074941.55         session:259   Tracker[HTTP, http://www.smartorrent.com:2710/ma7697h0anvl19yzswrrdadj57iyt35w/announce] Failed to send HTTP SCRAPE message: [Errno 111] Connection refused
ERROR   1437075002.36         session:259   Tracker[HTTP, http://www.smartorrent.com:2710/ma7697h0anvl19yzswrrdadj57iyt35w/announce] Failed to send HTTP SCRAPE message: [Errno 111] Connection refused
ERROR   1437075072.88         session:259   Tracker[HTTP, http://www.smartorrent.com:2710/ma7697h0anvl19yzswrrdadj57iyt35w/announce] Failed to send HTTP SCRAPE message: [Errno 111] Connection refused
ERROR   1437075133.73         session:259   Tracker[HTTP, http://www.smartorrent.com:2710/ma7697h0anvl19yzswrrdadj57iyt35w/announce] Failed to send HTTP SCRAPE message: [Errno 111] Connection refused
ERROR   1437075193.93         session:259   Tracker[HTTP, http://www.smartorrent.com:2710/ma7697h0anvl19yzswrrdadj57iyt35w/announce] Failed to send HTTP SCRAPE message: [Errno 111] Connection refused
ERROR   1437075254.46         session:259   Tracker[HTTP, http://www.smartorrent.com:2710/ma7697h0anvl19yzswrrdadj57iyt35w/announce] Failed to send HTTP SCRAPE message: [Errno 111] Connection refused
ERROR   1437075314.99         session:259   Tracker[HTTP, http://www.smartorrent.com:2710/ma7697h0anvl19yzswrrdadj57iyt35w/announce] Failed to send HTTP SCRAPE message: [Errno 111] Connection refused
ERROR   1437075375.49         session:259   Tracker[HTTP, http://www.smartorrent.com:2710/ma7697h0anvl19yzswrrdadj57iyt35w/announce] Failed to send HTTP SCRAPE message: [Errno 111] Connection refused
ERROR   1437075436.23         session:259   Tracker[HTTP, http://www.smartorrent.com:2710/ma7697h0anvl19yzswrrdadj57iyt35w/announce] Failed to send HTTP SCRAPE message: [Errno 111] Connection refused
ERROR   1437075496.80         session:259   Tracker[HTTP, http://www.smartorrent.com:2710/ma7697h0anvl19yzswrrdadj57iyt35w/announce] Failed to send HTTP SCRAPE message: [Errno 111] Connection refused
ERROR   1437075557.09         session:259   Tracker[HTTP, http://www.smartorrent.com:2710/ma7697h0anvl19yzswrrdadj57iyt35w/announce] Failed to send HTTP SCRAPE message: [Errno 111] Connection refused
ERROR   1437075618.08         session:259   Tracker[HTTP, http://www.smartorrent.com:2710/ma7697h0anvl19yzswrrdadj57iyt35w/announce] Failed to send HTTP SCRAPE message: [Errno 111] Connection refused
ERROR   1437075655.13       community:2577  already seen this indentifier in this batch, previous candidate {192.168.0.27:7759 24.69.63.0:7759} this one {192.168.0.27:7759 24.69.63.0:7759}
ERROR   1437075655.13       community:2577  already seen this indentifier in this batch, previous candidate {192.168.0.27:7759 24.69.63.0:7759} this one {192.168.0.27:7759 24.69.63.0:7759}
ERROR   1437075655.13       community:2577  already seen this indentifier in this batch, previous candidate {192.168.0.27:7759 24.69.63.0:7759} this one {192.168.0.27:7759 24.69.63.0:7759}
ERROR   1437075655.13       community:2577  already seen this indentifier in this batch, previous candidate {192.168.0.27:7759 24.69.63.0:7759} this one {192.168.0.27:7759 24.69.63.0:7759}
ERROR   1437075688.62         session:259   Tracker[HTTP, http://www.smartorrent.com:2710/ma7697h0anvl19yzswrrdadj57iyt35w/announce] Failed to send HTTP SCRAPE message: [Errno 111] Connection refused
ERROR   1437075748.75         session:259   Tracker[HTTP, http://www.smartorrent.com:2710/ma7697h0anvl19yzswrrdadj57iyt35w/announce] Failed to send HTTP SCRAPE message: [Errno 111] Connection refused
ERROR   1437075809.65         session:259   Tracker[HTTP, http://www.smartorrent.com:2710/ma7697h0anvl19yzswrrdadj57iyt35w/announce] Failed to send HTTP SCRAPE message: [Errno 111] Connection refused
ERROR   1437075870.53         session:259   Tracker[HTTP, http://www.smartorrent.com:2710/ma7697h0anvl19yzswrrdadj57iyt35w/announce] Failed to send HTTP SCRAPE message: [Errno 111] Connection refused
ERROR   1437075931.25         session:259   Tracker[HTTP, http://www.smartorrent.com:2710/ma7697h0anvl19yzswrrdadj57iyt35w/announce] Failed to send HTTP SCRAPE message: [Errno 111] Connection refused
ERROR   1437076001.69         session:259   Tracker[HTTP, http://www.smartorrent.com:2710/ma7697h0anvl19yzswrrdadj57iyt35w/announce] Failed to send HTTP SCRAPE message: [Errno 111] Connection refused
ERROR   1437076061.91         session:259   Tracker[HTTP, http://www.smartorrent.com:2710/ma7697h0anvl19yzswrrdadj57iyt35w/announce] Failed to send HTTP SCRAPE message: [Errno 111] Connection refused
ERROR   1437076122.26         session:259   Tracker[HTTP, http://www.smartorrent.com:2710/ma7697h0anvl19yzswrrdadj57iyt35w/announce] Failed to send HTTP SCRAPE message: [Errno 111] Connection refused
ERROR   1437076182.89         session:259   Tracker[HTTP, http://www.smartorrent.com:2710/ma7697h0anvl19yzswrrdadj57iyt35w/announce] Failed to send HTTP SCRAPE message: [Errno 111] Connection refused
ERROR   1437076243.50         session:259   Tracker[HTTP, http://www.smartorrent.com:2710/ma7697h0anvl19yzswrrdadj57iyt35w/announce] Failed to send HTTP SCRAPE message: [Errno 111] Connection refused
ERROR   1437076303.86         session:259   Tracker[HTTP, http://www.smartorrent.com:2710/ma7697h0anvl19yzswrrdadj57iyt35w/announce] Failed to send HTTP SCRAPE message: [Errno 111] Connection refused
ERROR   1437076364.60         session:259   Tracker[HTTP, http://www.smartorrent.com:2710/ma7697h0anvl19yzswrrdadj57iyt35w/announce] Failed to send HTTP SCRAPE message: [Errno 111] Connection refused
ERROR   1437076424.79         session:259   Tracker[HTTP, http://www.smartorrent.com:2710/ma7697h0anvl19yzswrrdadj57iyt35w/announce] Failed to send HTTP SCRAPE message: [Errno 111] Connection refused
ERROR   1437076485.25         session:259   Tracker[HTTP, http://www.smartorrent.com:2710/ma7697h0anvl19yzswrrdadj57iyt35w/announce] Failed to send HTTP SCRAPE message: [Errno 111] Connection refused
ERROR   1437076545.65         session:259   Tracker[HTTP, http://www.smartorrent.com:2710/ma7697h0anvl19yzswrrdadj57iyt35w/announce] Failed to send HTTP SCRAPE message: [Errno 111] Connection refused
ERROR   1437076606.54         session:259   Tracker[HTTP, http://www.smartorrent.com:2710/ma7697h0anvl19yzswrrdadj57iyt35w/announce] Failed to send HTTP SCRAPE message: [Errno 111] Connection refused
ERROR   1437076676.68         session:259   Tracker[HTTP, http://www.smartorrent.com:2710/ma7697h0anvl19yzswrrdadj57iyt35w/announce] Failed to send HTTP SCRAPE message: [Errno 111] Connection refused
```

**Comment by LipuFei** [comment]
@synctext Is this the latest `devel`?

**Comment by synctext** [comment]
yes

**Comment by whirm** [comment]
@LipuFei are you working on this?

**Comment by LipuFei** [comment]
@whirm Yes, I've made a fix for this #1566.
